decision_service/examples/rl_sim_cpp/person.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+#include <string>
+#include <unordered_map>
+class person {
```

Comment:
nit: new line after #pragma, and after includes

Reply:
:)  VSCode does a nice job with this but I added the space.